### PR TITLE
feat: onboards two JSS repos for large data sharing webapps

### PR DESCRIPTION
**Pull request by g-awmalik**

### CODE REVIEW

**Review by donmccasland — APPROVED** [review]

**Review by donmccasland — APPROVED** [review]

File: infra/terraform/test-org/org/locals.tf [hunk 1]
```diff
@@ -597,5 +597,19 @@ locals {
       owners      = ["dulacp"]
       topics      = join(",", [local.common_topics.da, local.common_topics.e2e], ["marketing"])
     },
+    {
+      name        = "terraform-large-data-sharing-java-webapp"
+      short_name  = "large-data-sharing-java-webapp"
+      org         = "GoogleCloudPlatform"
+      description = "Large data sharing Java web app"
```
Comment (bharathkkb) [inline infra/terraform/test-org/org/locals.tf:604]:
```suggestion
      description = "Deploys a large data sharing Java web app"
```

File: infra/terraform/test-org/org/locals.tf [hunk 2]
```diff
@@ -597,5 +597,19 @@ locals {
...
+      owners      = ["donmccasland"]
+    },
+    {
+      name        = "terraform-large-data-sharing-golang-webapp"
+      short_name  = "large-data-sharing-golang-webapp"
+      org         = "GoogleCloudPlatform"
+      description = "Large data sharing Golang web app"
```
Comment (bharathkkb) [inline infra/terraform/test-org/org/locals.tf:611]: nit: to align with other desc that starts with verb
```suggestion
      description = "Deploys a large data sharing Golang web app"
```

File: infra/terraform/test-org/org/locals.tf [hunk 3]
```diff
@@ -597,5 +597,19 @@ locals {
       owners      = ["dulacp"]
       topics      = join(",", [local.common_topics.da, local.common_topics.e2e], ["marketing"])
     },
+    {
+      name        = "terraform-large-data-sharing-java-webapp"
+      short_name  = "large-data-sharing-java-webapp"
```
[the remark this answers is not in the document]
Comment (g-awmalik) [reply infra/terraform/test-org/org/locals.tf:602]: Shortened.

**Review by apeabody — APPROVED** [review]
LGTM!

**Comment by apeabody** [comment]
Thanks @g-awmalik - We'll need admin for the repos to finish applying (create ci-triggers, etc).

**Comment by g-awmalik** [comment]
Applied.

**Pull request merged**